### ISSUE TITLE
Using 3-letter Codes

### ISSUE THREAD

**Issue by vikaskchoubey** [nic-delhi/AarogyaSetu_Android#279]
**../../src/main/res/values-mal:** 

For compatibility, should use 2-letter language codes when available; use ml instead of mal

**Comment by aravindvnair99** [comment]
I agree. `mal` is ISO 639-2 code. `ml` is ISO 639-1 code.  Should be `ml_IN` or `ml` as per [App resources overview | Android Developers](https://developer.android.com/guide/topics/resources/providing-resources.html#table2) following ISO 639-1, ISO 3166-1-alpha-2 and BCP 47 language tags.

Reference:

1. https://developer.android.com/reference/java/util/Locale.html
2. https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
